test/docfx.Test/build/BuildTest.cs

```diff
@@ -53,7 +54,7 @@ private static void VerifyFile(string file, string content)
             {
                 case ".json":
                     TestHelper.VerifyJsonContainEquals(
-                        JToken.Parse(content),
+                        JToken.Parse(content ?? "{}"),
```

Comment:
>content ?? "{}"), [](start = 37, length = 17)

May this cause confusion? It's not straightforward to interpret empty as an empty **object**. It can also be an empty array, empty string, or others.

Reply:
if the content is null, we assume it's a empty object, just for test :)

---
In reply to: [186912895](https://github.com/dotnet/docfx/pull/2737#discussion_r186912895) [](ancestors = 186912895)